src/main/java/com/dynatrace/prototype/domainModel/KeptnCloudEventValidator.java

```diff
@@ -0,0 +1,102 @@
+package com.dynatrace.prototype.domainModel;
+
+import java.util.HashMap;
+
+public class KeptnCloudEventValidator {
+    public static final String STAGE_NAME = "stageName";
+    public static final String SEQUENCE_NAME = "sequenceName";
+    public static final String EVENT_TYPE = "eventType";
+    public static final String TASK_NAME = "taskName";
+
+    private static final String TASK_PROJECT = "project";
+    private static final String TASK_SERVICE = "service";
+    private static final String TASK_CREATE = "create";
+
+    /**
+     * Parses the given String into a HashMap with the entries 'stageName', 'sequenceName' and 'eventType' if it is
+     * a valid event type.
+     * @param sequenceTriggeredEventType String to parse
+     * @return Filled HashMap if successful or else null
+     */
+    public static HashMap<String, String> parseSequenceEventType(String sequenceTriggeredEventType) {
+        HashMap<String, String> eventMetaData = null;
+        String[] parts = sequenceTriggeredEventType.split("\\.");
```

Comment:
FWIW `split()` has some weird behavior, I usually prefer the Guava implementation: https://guava.dev/releases/23.0/api/docs/com/google/common/base/Splitter.html 

With it, you don't need the logic of `isValidEventType(String[], int)`